### PR TITLE
Add support for Python 3.11

### LINKED ISSUE #27: Add support for Python 3.11
See also #25. It might make sense to wait for the end-of-life of Python 3.7, so that we don't have to manage too many python versions here...

### PR DESCRIPTION
**Pull request by joergbrech**
Closes #27

### CODE REVIEW

**Comment by joergbrech** [comment]
It seems Python 3.11 is not supported by conda-bld yet. So let's wait.

File: tigl3/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ source:
   git_tag: v{{ version }}
 
 build:
-  number: 2
+  number: 0
```
Comment (AntonReiswich) [inline tigl3/meta.yaml:12]: What is this number for?

Comment (joergbrech) [reply tigl3/meta.yaml:12]: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string You can use it to trigger a rebuild.

File: python-occ-7x/meta.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ source:
     - 0001-DLR-patch-to-support-c2-continous-coons-patches.patch
 
 build:
-  number: 5
+  number: 7
```
Comment (joergbrech) [inline python-occ-7x/meta.yaml:14]:
```suggestion
  number: 6
```

**Comment by joergbrech** [comment]
@AntonReiswich The build fails because the script https://github.com/DLR-SC/tigl-conda/blob/master/build_recipes.py#L65 only builds recipes that were changed in the last commit. tixi3 is changed in this PR, but not the latest commit, so it wasn't added to the list of recipes to build and it was missing. If we merge and squash this, we should be fine.

**Pull request merged**